acceptance/suites/tests/010-puppetserver-cli/subcommand/SERVER-297_common_behavior.rb

```diff
@@ -0,0 +1,33 @@
+require 'test/unit/assertions'
+require 'json'
+
+test_name "Puppetserver subcommand consolidated ENV handling tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+cli = "puppetserver"
```

Comment:
Do we need this comment and `cli` variable anymore?  Let's just inline it below and delete this cruft if so